server/api.js

```diff
@@ -552,11 +552,11 @@ SELECT bid.*, bidder.first_name, bidder.last_name, ba.attachment
 router.post("/bid/:bidId/status", async (req, res) => {
 	const bidId = parseInt(req.params.bidId, 10);
 	const status = req.body.status;
-	const validStatuses = ["Awarded", "Rejected", "Withdraw", "In review"];
+	const validStatuses = ["Awarded", "Rejected", "Withdrawn", "In review"];
 	const user = req.user;
 
 	if (!validStatuses.includes(status)) {
-		return res.status(400).send({ code: "INVALID_STATUS" });
+		return res.status(500).json({ code: "SERVER_ERROR" });
```

Comment:
A `400` is correct in this scenario, since the caller provides the status. Revert this back to what it was.